BE/src/main/java/todo3/codesquad/domain/Response.java

```diff
@@ -0,0 +1,21 @@
+package todo3.codesquad.domain;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+import java.util.List;
+
+@Getter @Setter @ToString
```

Comment:
한 가지 방법으로 통일해서 사용하는게 좋을 것 같아요.

```suggestion
@Getter
@Setter
@ToString
```